ansible/roles/zuul/files/zuul/layout.yaml

```diff
@@ -15,35 +14,54 @@ pipelines:
           comment: (?i)recheck(( (?:bug|lp)[\s#:]*(\d+))|( no bug))(\s+clean)?\s*$
         - event: comment-added
           comment: (?i)recheck windows$
+  - name: development
+    description: Pipeline used for Windows CI development. It should always work in silent mode.
+    manager: IndependentPipelineManager
+    precedence: low
+    trigger:
+      gerrit:
+        - event: comment-added
+          comment: (?i)recheck windows-exp$
```

Comment:
I think we should use: "recheck windows-dev"

Reply:
Perhaps `(?:-| )` so one doesn't have to remember whether it was `-` or space.
I'd also add a `\s*` before `$` in each comment, for the trailing whitespace to be ignored.